test/lib/y2users/config_merger_test.rb

```diff
@@ -194,10 +194,10 @@ def lhs_group(name)
           expect(lhs.groups.map(&:name)).to contain_exactly("test1", "test2", "test3")
         end
 
-        it "updates the lhs group with the data from the corresponding rhs" do
+        it "updates the lhs group with the data from the corresponding rhs group except gid" do
           subject.merge
 
-          expect(lhs_group("test1")).to eq(group1)
+          expect(lhs_group("test1").gid).to eq(100)
```

Comment:
this is probably left over from my commit as it should not change merger behavior, so test should fail.